pkgs/by-name/zu/zuban/package.nix

```
@@ -0,0 +1,47 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+  versionCheckHook,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "zuban";
+
+  version = "0.0.21";
+
+  src = fetchFromGitHub {
+    owner = "zubanls";
+    repo = "zuban";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-llAJwqJmOtauA7pi7dYZhVH0yFBNN65q2y8ecc9hAJY=";
+  };
+
+  buildAndTestSubdir = "crates/zuban";
+
+  cargoHash = "sha256-sZFc+kjerR7a6JzVb0zQXCfYuJVNnoYUihgLd3qrDLM=";
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgram = "${placeholder "out"}/bin/zuban";
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "A Mypy-compatible Python Language Server built in Rust";
```

```suggestion
    description = "Mypy-compatible Python Language Server built in Rust";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description